lib/content-services/src/lib/search/components/search-sorting-picker/search-sorting-picker.component.spec.ts

```diff
@@ -53,7 +53,9 @@ describe('SearchSortingPickerComponent', () => {
                 <any> { id: 'cat1', enabled: true }
             ]
         };
-        queryBuilder = new SearchQueryBuilderService(buildConfig(config), null);
+        const alfrescoApiService = TestBed.inject(AlfrescoApiService);
+
+        queryBuilder = new SearchQueryBuilderService(buildConfig(config), alfrescoApiService);
```

Comment:
why do we need to initialize the service? we should just get the instance with TestBed and mock the methods ...

Reply:
was already like that I just made it works, I suggest to refactor it in the second instance because al those private initialate API should actually come from services IMO